src/Schema/Name.php

```diff
@@ -12,8 +12,7 @@ interface Name
     /**
      * Returns the string representation of the name.
      *
-     * The consumers of this method should not rely on a specific return value. It should be used only for diagnostic
-     * purposes.
+     * If passed to the corresponding parser, the name should be parsed back to an equivalent object.
```

Comment:
This contract enables this expression: https://github.com/doctrine/dbal/blob/7a25c5d5d61820e47097e230162b10278a158b63/src/Schema/Table.php#L867